roles/mount_discovery_iso_for_pxe/defaults/main.yml

```diff
@@ -1,6 +1,6 @@
 ---
 # defaults file for generate_discovery_iso
-tftp_directory: "/var/lib/tftpboot"
+tftp_directory: "{{ hostvars['httpd_pxe_host']['httpd_port_for_pxe'] }}"
```

Comment:
If you get the `TFTP_ROOT` var from the playbook, do you need this variable? You can use `TFTP_ROOT` instead